static/src/javascripts/projects/common/modules/identity/consents.js

```diff
@@ -153,49 +95,45 @@ const buildFormDataForFields = (
 const getInputFields = (labelEl: HTMLElement): Promise<NodeList<HTMLElement>> =>
     fastdom.read(() => labelEl.querySelectorAll('*[name][value]'));
 
-const resetUnsubscribeFromAll = buttonEl => {
-    fastdom.write(() => {
-        $(buttonEl).removeClass(
-            'email-unsubscribe--confirm js-confirm-unsubscribe'
+const resetUnsubscribeFromAll = (buttonEl: HTMLButtonElement) =>
+    fastdom
+        .read(() => [
+            [...document.querySelectorAll('.js-unsubscribe--confirm')],
+            [...document.querySelectorAll('.js-unsubscribe--basic')],
+        ])
+        .then(([confirmEls, basicEls]) =>
+            fastdom.write(() => {
```

Comment:
Could you add a comment, that we could simplify that once `classList.remove` with multiple classnames is fixed?